tests/common.py

```diff
@@ -33,7 +33,8 @@
     "validate_const_node",
     "group_nodes_by_type",
     "test_ms_domain",
-    "check_node_domain"
+    "check_node_domain",
+    "check_thresholded_relu_count"
```

Comment:
not really need to add a method, just use check_op_count in test_thresholded_relu